### ISSUE TITLE
Links in image captions for Cartoon/picture tone are blending with the background like animals in the jungle

### ISSUE THREAD

**Issue by paperboyo** [guardian/frontend#12606]
Hello,

All browsers, OSes.

URL: http://www.theguardian.com/books/picture/2016/apr/19/re-illustrating-the-jungle-book-monkey-kidnapping-by-sarah-mcintyre-and-philip-reeve

The links in caption fail all accessibility so much that I won't even check by how much:

![picture 13](https://cloud.githubusercontent.com/assets/6032869/14648198/fe6b95fa-0658-11e6-8d24-82e5290ec022.png)

They would better be styled similarly to links in captions for the Gallery tone:

![picture 14](https://cloud.githubusercontent.com/assets/6032869/14648215/11d2093a-0659-11e6-9ca0-5f76149162fd.png)

Regards
Mateusz

**Comment by katebee** [comment]
Need to add the required styles to links inside the figcaption element, however, they should only be applied if the figcaption is a child of an element with the class .tonal--tone-media (which adds a dark background).

Here is a link that we **do not** want to affect that uses the same figcaption class, and also has a link:
http://www.theguardian.com/world/2016/jan/04/sunni-allies-join-saudi-arabia-in-ceasing-diplomatic-ties-with-iran

**Comment by michaelwmcnamara** [comment]
fixed
